client/components/Forms/EventForm.js

```diff
@@ -47,7 +48,9 @@ class EventForm extends Component {
     formData: this.props.formData,
     errors: this.errors,
     category: '',
-    categoryError: false,
+    categoryNoError: true,
```

Comment:
Nice work @chibujax.PR looks good. One thing though. categoryError, timezoneError feels more natural. Any reason behind categoryNoError and timezoneError ?